backend/models/storage.go

```
@@ -1218,56 +1569,112 @@ func (db *Database) CreateGithubAppInstallation(installationId int64, githubAppI
-	log.Printf("GithubAppInstallation (installationId: %v, githubAppId: %v, login: %v, accountId: %v, repoFullName: %v) has been created successfully\n", installationId, githubAppId, login, accountId, repoFullName)
+	slog.Info("GitHub app installation created successfully",
+		slog.Group("github",
+			"installationId", installationId,
+			"githubAppId", githubAppId,
+			"login", login,
+			"accountId", accountId,
+			"repoFullName", repoFullName))
 	return installation, nil
 }
 
 func validateDiggerConfigYaml(configYaml string) (*configuration.DiggerConfig, error) {
```

golangci-lint (1.64.8)

func `validateDiggerConfigYaml` is unused

(unused)